physical/raft/raft.go

```diff
@@ -474,9 +480,55 @@ func (b *RaftBackend) Close() error {
 		return err
 	}
 
+	close(b.statsDoneCh)
```

Comment:
If you don't wind up setting up metrics in NewRaftBackend, this might need an `if`.

Reply:
Ah yes, good call.